### PR TITLE
refactorings

### PR DESCRIPTION
**Pull request by swiknaba**

### CODE REVIEW

File: iam/iam-policy-for-ecs/core/README.md [hunk 1]
```diff
@@ -19,45 +19,44 @@ locals {
...
-      environment = "sandbox"
-      region      = "eu-central-1"
-    },
-    {
-      name        = "facebook",
-      environment = "load-testing"
-      region      = "eu-central-1"
-    },
-  ]
+  username       = local.users["gh-user"]["iam"]
```
Comment (swiknaba) [inline iam/iam-policy-for-ecs/core/README.md:57]: do we need to iterate over `local.uses` and creat that policy for each user?

File: iam/iam-policy-for-ecs/core/main.tf [hunk 2]
```diff
@@ -1,51 +1,55 @@
...
-    [
-      for name in local.staging_read_access_project_list : "${name}-staging"
-    ],
-    [
-      for name in local.production_read_access_project_list : "${name}-production"
+  # [
+  #   {
+  #       "name"        = "facebook-production"
+  #       "env"         = "production"
+  #       "project_tag" = "production-developer-access-projects"
```
Comment (swiknaba) [inline iam/iam-policy-for-ecs/core/main.tf:6]:
```suggestion
```

same for the other examples

File: iam/iam-policy-for-ecs/core/README.md [hunk 3]
```diff
@@ -19,45 +19,44 @@ locals {
...
-    staging-developer-access-projects    = try(each.value["staging-developer-access-projects"], "")
-    staging-admin-access-projects        = try(each.value["staging-admin-access-projects"], "")
-    production-developer-access-projects = try(each.value["production-developer-access-projects"], "")
-    production-admin-access-projects     = try(each.value["production-admin-access-projects"], "")
+
+    staging-developer-access-projects    = join(":", try(each.value["developer"]["staging"], []))
+    staging-admin-access-projects        = join(":", try(each.value["admin"]["staging"], []))
+    production-developer-access-projects = join(":", try(each.value["developer"]["production"], []))
+    production-admin-access-projects     = join(":", try(each.value["admin"]["production"], []))
+
```
Comment (swiknaba) [inline iam/iam-policy-for-ecs/core/README.md:49]:
```suggestion
```

File: iam/iam-policy-for-ecs/core/README.md (hunk 1 above)
Comment (samkahchiin) [reply iam/iam-policy-for-ecs/core/README.md:57]: Yes. because the policy is user-specific

**Pull request merged**